src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -374,7 +374,7 @@ def build_group_attachment(
     identity: Identity | None = None,
     actions: Sequence[MessageAction] | None = None,
     rules: list[Rule] | None = None,
-    link_to_event: bool = False,
+    link_to_event: bool = True,
```

Comment:
Instead of changing the default behavior, what about passing `link_to_event=True` in `notify_action.py`'s `send_notification`'s usage of this function?